### PR TITLE
Update documentation to v1.0.0.

### LINKED ISSUE #42: Docs: MathJax not loaded -> math not rendered
**Describe the bug**
In the rendered docs at https://psu-inversion.github.io/atmospheric-inverse-methods-for-flux-optimization/
math does not display correctly because MathJax is not being loaded.

I tested inserting this snippet into a page's `<head>`
```
  <!-- MathJax Support -->
  <script async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
```
and it worked then.

Maybe you could set `mathjax_path` explicitly in your conf?

### PR DESCRIPTION
**Pull request by DWesl**
## Description

Hopefully this fixes #42.

## Motivation and Context

This brings the published version of the documentation up-to-date with the code version, which is useful given the recent release.

## How Has This Been Tested?

I checked locally to make sure MathJax renders

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [X] I give permission for my code to be distributed under the existing license

### CODE REVIEW

File: _modules/atmos_flux_inversion.html [hunk 1]
```diff
@@ -15,6 +15,8 @@
     <script type="text/javascript" src="../_static/underscore.js"></script>
     <script type="text/javascript" src="../_static/doctools.js"></script>
     <script type="text/javascript" src="../_static/language_data.js"></script>
+    <script async="async" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
+    <script type="text/x-mathjax-config">MathJax.Hub.Config({"extensions": ["tex2jax.js"], "jax": ["input/TeX", "output/HTML-CSS"], "TeX": {"extensions": ["AMSmath.js"]}})</script>
```
Comment (zmoon) [inline _modules/atmos_flux_inversion.html]: example where MathJax config is specified twice in the HTML output

Comment (DWesl) [reply _modules/atmos_flux_inversion.html]: I see where the `MathJax.Hub.Config(...)` line comes from, but I don't know where the `config=TeX-AMS-MML_HTMLorMML` part comes from.  I'm suspecting that's added by `sphinx.ext..mathjax` itself.  I'll look into how to change that.

Comment (zmoon) [reply _modules/atmos_flux_inversion.html]: yeah it seems like that might be the default `mathjax_path` setting
https://github.com/sphinx-doc/sphinx/blob/a721631a541db298f7f9807c9db8433e928cb20b/sphinx/ext/mathjax.py#L98

Comment (zmoon) [reply _modules/atmos_flux_inversion.html]: so I guess workaround could be to set `mathjax_path` in your conf to `'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js'`

Comment (zmoon) [reply _modules/atmos_flux_inversion.html]: or remove your `mathjax_config`, or just not worry about it

Comment (DWesl) [reply _modules/atmos_flux_inversion.html]: Updating to latest Sphinx leaves the duplicate configuration.  I dropped the manual MathJax configuration, which leaves just the first line and seems to display the equations.

**Pull request merged**